content/how-to/fabric/fabric-ticketing/index.md

```diff
@@ -0,0 +1,153 @@
+---
+permalink: "fabric-ticketing"
+audit_date:
+type: article
+title: "Fabric ticketing"
+created_date: '2022-03-04'
+created_by:
+last_modified_date:
+last_modified_by:
+product:
+product_url:
+---
+# Introduction
+Rackspace Technology uses a new ticketing application as part of its Fabric branding. The ticketing application follows ITSM and ITIL change management standards by allowing customers to designate a category for the ticket.
+
+## Intended Audience for Docs
+This documentation is intended for assisting customers with the new Fabric Ticketing Application. 
+
+*This documentation describes the user experience (aka. BPiT-Business Process Input Template) to support customer and assist customer with application issues related to this portal.*
+
+## Access to Fabric Ticketing Application
+You can access the ticketing application by entering login credentials. This page can be reached from any of the current customer portal URLs as listed below.
+* https://my.rackspace.com/ 
+* https://manage.rackspace.com/ 
+* https://portal.rackspace.com/ 
+
+Enter your email address and password to login into the portal. 
+
+<img width="540" alt="FT1" src="https://user-images.githubusercontent.com/98875667/156165005-98fc3bb1-c56b-47dd-b1e9-ec2c31403728.png">
+
+### Global Dashboard Overview
+
+<img width="945" alt="FT6" src="https://user-images.githubusercontent.com/98875667/156172786-71f7f0d1-5193-4fc4-a8f3-e4ef9b35a8b9.png">
+
+For all multi-cloud and non-dedicated users, the ticket navigation option will take you to the new ticketing application as shown in the above image.
+
+The global dashboard view has tabs as listed below:
+* Tickets
+* Support
+* Billing
+* Account
+
+#### Tickets
+
+<img width="177" alt="FT4" src="https://user-images.githubusercontent.com/98875667/156175991-029922e0-4e20-4b08-af23-1088e2e62d1b.png">
+
+The user must be granted ticketing permissions for the ticket navigation items to be allowed.
+
+### Application Overview
+
+<img width="950" alt="FT7" src="https://user-images.githubusercontent.com/98875667/156180814-ee83a84d-ece0-41ff-b6fd-4e1019a99ad6.png">
+
+**Ticket List Page**: Displays the list of tickets created. The list displays either the most recently created ticket or last modified ticket at the top. 
+
+#### **Left Side Ticket Panel**: 
+
+<img width="302" alt="FT9" src="https://user-images.githubusercontent.com/98875667/156182984-cf86867f-a906-45bd-9e37-e0a8daba6c67.png">
+
+1. Create Ticket: User can create new ticket using this option. Details of how new ticket is created is detailed in upcoming sections. 
+2. Search: User can search the required tickets. 
```

Comment:
we should update to include more details like this..
`Search: User can utilize the field to search by Ticket ID, Subject, and for text used in comments.` @samatha-kankipati Am I missing anything on this?